bin/update-abq-formulae

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ "$#" -ne 1 ]]; then
+  >&2 echo "Usage: $0 X.X.X"
+  exit 2
+fi
+
+oldVersion=$(grep -E "version \"\d+\.\d+\.\d+\"" ./Formula/abq.rb | sed -E "s/.*version \"(.+)\".*/\1/")
+newVersion=$1
+
+if [[ "${oldVersion}" == "${newVersion}" ]]; then
+  echo "Exiting early... the version is already ${newVersion}"
+  exit 0
+fi
+
+mkdir -p tmp
+
+function getSHA() {
+  url=$1
+  expectedVersion=$2
+  fake_user_agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/602.4.8 (KHTML, like Gecko) Version/10.0.3 Safari/602.4.8"
```

Comment:
This is what `user_agent: :fake` does in homebrew. 

https://github.com/Homebrew/brew/blob/b3f7c6f7a5f6cc8fe87aeac112e0ba96917ae0f8/Library/Homebrew/utils/curl.rb#L84C1-L85
https://github.com/Homebrew/brew/blob/b3f7c6f7a5f6cc8fe87aeac112e0ba96917ae0f8/Library/Homebrew/global.rb#L48-L50